imgix/urlhelper.py

```diff
@@ -14,6 +14,42 @@
 
 
 class UrlHelper(object):
+    """
+    Helper class to create single domain Imgix URLs. Example:
+
+      >>> str(UrlHelper('demos.imgix.net', '/bridge.png', opts={'w': 100}))
+      https://demos.imgix.net/bridge.png?w=100
+
+    Parameters
+    ----------
+    domain : str
+    path : str
+    scheme : { 'https', 'http' }
+    sign_key : str or None
+        Use the key to generate signed image URLs. URLs are not signed by
+        default. (default `None`)
+    sign_mode : {`SIGNATURE_MODE_QUERY`, `SIGNATURE_MODE_PATH`}
```

Comment:
I'm not sure what `SIGNATURE_MODE_PATH` would do, and I don't like the idea of maintaining a single-option configuration variable just for the sake of "future support". Any idea what this is about?

Reply:
I couldn't find any reference to `SIGNATURE_MODE_PATH` in any other libraries. I will go ahead and remove it (only) from the documentation for the time being.